packages/wdio-browserstack-service/src/Percy/Percy.ts

```diff
@@ -114,24 +123,43 @@ class Percy {
 
     async fetchPercyToken() {
         const projectName = this.#projectName
-
+        let data = {
+            enabled: this.percyAutoEnabled,
+            token: null,
+            percyCaptureMode: this.percyCaptureMode
+        }
         try {
             const type = this.#isApp ? 'app' : 'automate'
-            const response = await nodeRequest(
-                'GET',
-                `api/app_percy/get_project_token?name=${projectName}&type=${type}`,
+            let query = 'api/app_percy/get_project_token?'
+            if (projectName) {
+                query += `name=${projectName}&`
+            }
+            if (type) {
+                query += `type=${type}&`
+            }
+            if (this.#options.percy !== undefined) {
+                query += `percy=${this.#options.percy}&`
+            }
+            if (this.#options.percyCaptureMode) {
+                query += `percy_capture_mode=${this.#options.percyCaptureMode}`
```

Comment:
If user doesn't pass any capture mode. Sdk shouldn't set any default capture mode. Send undefined in that case.

Reply:
Yes, we are not sending any default capture mode, if user has not passed, then `this.#options.percyCaptureMode` is undefined